### PR TITLE
docs: Use Select component in ThemeSelect

### LINKED ISSUE #1223: Use our own Select component in marigold docs ThemeSwitch
## Description

Use our own Select component in marigold docs ThemeSwitch instead of <Box as="select">

## Context

We want to use in marigold docs our own written components to test them all in production.

## Consequences

We get a product where we use all our written components.

### PR DESCRIPTION
**Pull request by ti10le**
closes #1223

### CODE REVIEW

File: packages/components/src/Select/Select.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ export type SelectProps = {
   disabled?: boolean;
   required?: boolean;
   error?: string;
+  width?: string;
```
Comment (viktoria-schwarz) [inline packages/components/src/Select/Select.tsx:27]: I would like to restrict the type here more, like responsive style values?

File: docs/content/components/select.mdx [hunk 2]
```diff
@@ -20,6 +20,7 @@ Dropdown for selecting an item among different options.
 | `disabledKeys` (optional)       | [Iterable](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols)`<Key>` |         |
 | `required` (optional)           | `boolean`                                                                                                |         |
 | `error` (optional)              | `string`                                                                                                 |         |
+| `width` (optional)              | `string`                                                                                                 |         |
```
Comment (viktoria-schwarz) [inline docs/content/components/select.mdx:23]: can you add the new prop to storybook aswell?

File: packages/components/src/Select/Select.tsx (hunk 1 above)
Comment (ti10le) [reply packages/components/src/Select/Select.tsx:27]: Yes I thought about this. Like this at menuWidth: https://react-spectrum.adobe.com/react-spectrum/Picker.html#props
but we have not yet, right? We could add it to the BaseTheme?

Comment (viktoria-schwarz) [reply packages/components/src/Select/Select.tsx:27]: I think we have ResponsiveStyleValue which is used in other components as well :)

Comment (ti10le) [reply packages/components/src/Select/Select.tsx:27]: oh sure, I did not think of that just now 🙈

**Review by viktoria-schwarz — APPROVED** [review]

**Pull request merged**